### ISSUE TITLE
Permanently crashes whole site with Remove theme button !

### ISSUE THREAD

**Issue by NorthDecoder** [VividVenturesLLC/theme_supermint#4]
In an attempt to test the function of the `Remove` Supermint theme button, I found that button crashes the website with no recovery!

* c5 version 8.4.2
* supermint3 at commit 6325b6b5b4c5210d07727eaa9fd30f97589d086d 

### Steps to produce
1. Fresh install c5 8.4.2
2. Install Supermint files by copying from development directory external to c5_842 directory.
```
mkdir ~/public_html/c5_842/public/packages/theme_supermint
cp * ~/public_html/c5_842/public/packages/theme_supermint
```
3. In dashboard Activate theme. Poke around in theme dashboard.
4. In Dashboard->Pages & Themes `Activate` an alternate theme Elemental.
5. On the themes page click the Supermint `Remove` button 

### Error message:
```
An unexpected error occurred.
Call to a member function getPackageHandle() on null
```

In browser right click and copy link location over remove button returns:
```
http://my.website.com/c5_842/public/index.php/dashboard/pages/themes/remove/2/1535035438%3Afaabe1ff5a49e616f104b166c27db751
```

**Comment by NorthDecoder** [comment]
@weyboat replied on Aug 14, 2018 at 2:56 pm in the forum [:link:](https://www.concrete5.org/community/forums/themes/supermint-error-call-to-a-member-function-getpackagehandle-on-nu/#938612)

>Okay I just tested this myself on Version 8.4..
What I found was if you activate the elemental theme and then go back to themes page and click 'remove' on the supermint theme the error is thrown as per your post..

>However, if you use the 'Uninstall' feature in the 'extend concrete5' page, it removes supermint without an error.

**Comment by NorthDecoder** [comment]
Following the steps to reproduce above and with sitewide
debugging messages turned on from within the dashboard.

The last message, and first message with error noted:
```
/public/packages/theme_supermint/controllers/tools/preset_colors.php:32 
```
![first-error-delete-sm3-theme](https://user-images.githubusercontent.com/9598008/44746867-7b11b180-aac0-11e8-8d7b-3f96c4c75309.JPG)

**Comment by NorthDecoder** [comment]
Just an educated guess...
In commit fbcc5a9
In file controllers/tools/override_css.php at line eighteen
this was changed to:
```
$c = Page::getByID($this->request->query->get('cID'));
```
So I guess that this same change should be made in `preset_colors.php` line 28.
FROM:
```
        $c = $_REQUEST['cID'] ? Page::getByID($_REQUEST['cID']) : Page::getCurrentPage();
```
TO:
```
         $c = $_REQUEST['cID'] ? Page::getByID($this->request->query->get('cID')) : Page::getCu$
```

I have no idea why there is a difference.

**Comment by NorthDecoder** [comment]
@weyboat , @MichaelMaar
I think I have fixed the bug that was causing a fatal website crash during `remove` of the supermint theme, by ignoring the page_theme which is not available during deletion.

Should you need it, the fix is tagged and pushed as release [v3.4.1.4](https://github.com/VividVenturesLLC/theme_supermint/releases/tag/v3.4.1.4)
